packages/desktop-client/src/components/spreadsheet/CellValue.js

```diff
@@ -17,7 +17,6 @@ function CellValue({ binding, type, formatter, style, getStyle, debug }) {
               style,
               getStyle && getStyle(value),
             ]}
-            numberOfLines={1}
```

Comment:
Should this component be changed to `TextOneLine`?

Reply:
Don't think so. Double-checked it in the transaction table and it seems to work OK without changing to `TextOneLine` (so basically it currently has the same logic as it has had since Actual going open source.. minus the unused prop).